src/main/java/frc/robot/commands/ClimbEnable.java

```diff
@@ -28,7 +28,7 @@ public void initialize() {
     m_subsystem.resetTargetedHeight();
 
     if (m_subsystem.getclimbingenable()) {
-      m_drivesubsystem.setDriveBaseSpeed(0.40);
+      m_drivesubsystem.setDriveBaseSpeed(0.6);
```

Comment:
How about we make this a constant so its easy to find where to tweek it?

Reply:
Done